pyproject.toml

```diff
@@ -64,7 +64,7 @@ pytest-sugar = "^1.0.0"
 pandas-stubs = "^2.2.1.240316"
 types-networkx = "^3.2.1.20240331"
 requests-mock = "^1.12.1"
-testcontainers =  "^4.4.1"
+testcontainers =  "4.7.2"
```

Comment:
There's a problem with the newest version which produces a fuseki container that looks healthy, but refuses any incoming HTTP connections. Once this PR is merged, I have to investigate this, so that later, we can also upgrade testcontainers.

Reply:
EDIT: It seems as if testcontainers >= 4.8.0 is buggy. My investigations showed a similar behavior than described in https://github.com/testcontainers/testcontainers-python/issues/633. But I'm unable to work around it, so I'm afraid we will have to stick with testcontainers == 4.7.2.